app/screens/more_channels/more_channels.js

```diff
@@ -4,7 +4,7 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {intlShape} from 'react-intl';
-import {Platform, View} from 'react-native';
+import {Platform, View, Picker} from 'react-native';
```

Comment:
I don;'t think the picker here is what we want, take a look at the `utils/bottom_sheet` component

Reply:
I've replaced it with the BottomSheet component - this is my first adventure in React Native, so I'm a little bit awkward with it. Any advice on how to style the Text element I've now added better? I've just used inline styles for now, but I'm sure there's a more elegant solution available!

Reply:
You can define a `StyleSheet` in the component